packages/client/Atmosphere.ts

```diff
@@ -278,6 +278,22 @@ export default class Atmosphere extends Environment {
       }
     }
     const transport = uploadables ? this.baseHTTPTransport : this.transport
+    const errorCheckerSink: Sink<any> | undefined = sink
+      ? {
+          next: (value) => {
+            if (value.errors) {
+              throw new Error(value.errors[0]?.message)
```

Comment:
-2 I'd like to work together to figure out another way to do this. A test case where this wouldn't work is a poker meeting where a user can't access a jira issue. The meeting is good. The phases are good. 99% of the data is good except the 1 jira issue that couldn't get fetched. If we throw here, then would the whole meeting object get thrown out? (I didn't test this specific use case, so I could be wrong!).